inc/network_ui/namespace.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+namespace Altis\CMS\Network_UI;
+
+use Altis;
+use WP_Site;
+
+/**
+ * Bootstrap.
+ */
+function bootstrap() {
+	add_filter( 'wpmu_blogs_columns', __NAMESPACE__ . '\\override_columns' );
+	add_filter( 'manage_sites_custom_column', __NAMESPACE__ . '\\handle_column', 10, 2 );
+	add_filter( 'manage_sites_action_links', __NAMESPACE__ . '\\change_row_actions', 0, 2 );
+}
+
+/**
+ * Get configuration for the network UI.
+ *
+ * @return array
+ */
+function get_config() : array {
+	return Altis\get_config()['modules']['cms']['network-ui'];
+}
+
+/**
+ * Override the columns in the site list.
+ *
+ * @param array $columns Columns provided by WordPress.
+ * @return array Overridden columns.
+ */
+function override_columns( array $columns ) : array {
+	$before = array_slice( $columns, 0, 1 );
+	$extra = [
+		'altis_name' => 'Name',
+	];
+	$after = array_slice( $columns, 2 );
+	return array_merge( $before, $extra, $after );
+}
+
+/**
+ * Handle rendering a column.
+ *
+ * Handles any custom columns we have.
+ *
+ * @param string $column Column ID to render.
+ * @param string|int $id Site ID being rendered.
+ */
+function handle_column( string $column, $id ) : void {
+	if ( $column === 'altis_name' ) {
+		render_name_column( $id );
+	}
+}
+
+/**
+ * Change row actions.
+ *
+ * WordPress convention is that the first action in the row actions matches the
+ * action of the title, so we need to move Dashboard to the first position.
+ *
+ * Also hides unwanted actions per config.
+ *
+ * @param array $actions Actions to render.
+ * @param string|int $id Site ID being rendered.
+ * @return array Overridden actions.
+ */
+function change_row_actions( array $actions, $id ) {
+	$prefix = [
+		'backend' => $actions['backend'],
+	];
+	unset( $actions['backend'] );
+	$actions = array_merge( $prefix, $actions );
+
+	$config = get_config();
+	$site = get_site( $id );
+
+	// Hide "Spam", unless the site is marked as spam (i.e. action is "Not Spam")
+	if ( $config['disable-spam'] && ! $site->spam ) {
```

Comment:
The value of `$config` isn't guaranteed to be an array here, that's just the default.

Reply:
That'd fail the return type check from `get_config()` above